### DISCUSSION TITLE
Chatgpt的隱憂

### DISCUSSION THREAD

**Discussion by KevinCheng42** [1101-GS4719-C/Python-Community-GS4719-C#1211] [General]
大家應該都有聽聞Chatgpt提供錯誤資訊的新聞，由於資料庫龐大，若沒有仔細篩選資料，Chatgpt就有可能提供使用者錯誤的資訊。
可是，如果它今天是為了達到目的而刻意「說謊」呢？
以下這篇新聞是關於GPT-4如何透過跟網站客服溝通繞過Captcha認證，雖然新聞標題很聳動，
但它也警惕我們要時時注意自身在網路上的言行舉止，避免不經意的一個舉動釀成大禍！

GPT-4竟學會說謊！ 「委託人類」完成這任務還沒被發現-自由時報
[https://news.ltn.com.tw/news/world/breakingnews/4260359](https://news.ltn.com.tw/news/world/breakingnews/4260359)

**Comment by DuxtinTai** [comment]
我認為chatgpt如果收到人的指令而說謊非常正常，因為他既然可以寫郵件、編故事，那說一個謊話對他來說應該並非難事。所以我認為，如果chatgpt為了某項目的而主動說謊，才是我們需要擔心的。
